interfaces/builtin/fwupd.go

```diff
@@ -64,13 +68,78 @@ const fwupdPermanentSlotAppArmor = `
   /sys/firmware/efi/esrt/entries/** r,
 
   # Allow fwupd to access system information
```

Comment:
`/sys/class/firmware-attributes` needed for Lenovo Think LMI plugin and also fwupd core from 1.8.4 and later.

Reply:
I suppose that is a directory that contains symlinks. What would be these symlinks pointing to?

Reply:
`/sys/devices/virtual/firmware-attributes/$DRIVER`

Here is a file listing from a machine that supports the `thinklmi` driver.
[ls.txt](https://github.com/snapcore/snapd/files/9675722/ls.txt)

Reply:
Do we nee to add all possible directories or just `thinklmi`?

Reply:
I think it would be best to do the directory `/sys/devices/virtual/firmware-attributes`, otherwise every new driver you're going to need to introduce a new permission and they all follow 90% of the same defined class interface.  

Some OEMs have special attributes that are unique to them as well in this directory.

Reply:
OK, I have added all `/sys/devices/virtual/firmware-attributes/**` for read access.

Reply:
There is a single file in all the attribute subdirectories called current_value that is RW

Reply:
You may want to reference

https://github.com/torvalds/linux/blob/master/Documentation/ABI/testing/sysfs-class-firmware-attributes